noir-projects/noir-contracts/contracts/avm_test_contract/src/main.nr

```diff
@@ -567,3 +567,15 @@ contract AvmTest {
         let _ = nested_static_call_to_add(inputs, 1, 2);
     }
 }
+
+mod tests {
+    use super::AvmTest;
+    use aztec::context::inputs::public_context_inputs::PublicContextInputs;
+
+    #[test]
+    fn contract_has_public_dispatch() {
+        let selector = 0;
+        let inputs = PublicContextInputs::empty();
+        AvmTest::public_dispatch(inputs, selector);
+    }
```

Comment:
This test will probably fail and we should remove it, but I left it here to check that the code type-checks.

Reply:
If you know your way in the aztec repo you could add a test here: https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/simulator/src/avm/avm_simulator.test.ts

That calls e.g., add_args_return via the dispatch function.

Otherwise I can commit to this branch and add it myself once the publiccontextinputs are gone.